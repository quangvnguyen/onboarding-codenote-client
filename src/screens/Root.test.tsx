import React from 'react';
import { shallow } from 'enzyme';
import Root from './Root';

test('renders without crashing', () => {
  const wrapper = shallow(
    <Root />
  );
  expect(wrapper).toMatchSnapshot();
});
