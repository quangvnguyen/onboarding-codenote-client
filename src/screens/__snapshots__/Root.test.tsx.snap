// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Fragment>
  <AppliedRoute
    component={[Function]}
  />
  <Switch>
    <AppliedRoute
      component={[Function]}
      exact={true}
      path="/"
    />
    <Connect(UnauthenticatedRoute)
      component={[Function]}
      exact={true}
      path="/signup"
    />
    <Connect(UnauthenticatedRoute)
      component={[Function]}
      exact={true}
      path="/login"
    />
    <Connect(AuthenticatedRoute)
      component={[Function]}
      exact={true}
      path="/notes/new"
    />
    <Connect(AuthenticatedRoute)
      component={[Function]}
      exact={true}
      path="/notes/:id"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</Fragment>
`;
